tests/test_observatory.py

```diff
@@ -119,11 +119,17 @@ def test_min_max_antpos(bm):
 
 def test_from_uvdata(bm):
     uv = pyuvdata.UVData()
-    uv.antenna_positions = np.array([[0, 0, 0], [0, 1, 0], [1, 0, 0], [40, 0, 40]]) * units.m
-    uv.telescope_location = [x.value for x in EarthLocation.from_geodetic(0, 0).to_geocentric()]
+    uv.telescope.antenna_positions = (
+        np.array([[0, 0, 0], [0, 1, 0], [1, 0, 0], [40, 0, 40]]) * units.m
+    )
+    uv.telescope.location = [x.value for x in EarthLocation.from_geodetic(0, 0).to_geocentric()]
```

Comment:
I think this line is the bad one -- the location is now just an `EarthLocation` object, so it's much simpler: `uv.telescope.location = EarthLocation(from_geodetic(0,0)`